games-fps/redeclipse/redeclipse-1.6.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop
+
+DESCRIPTION="First-person ego-shooter, built as a total conversion of Cube Engine 2"
+HOMEPAGE="http://www.redeclipse.net/"
+SRC_URI="https://github.com/red-eclipse/base/releases/download/v${PV}/${PN}_${PV}_nix.tar.bz2"
+
+# According to doc/license.txt file
+LICENSE="HPND ZLIB CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dedicated"
```

Comment:
Looks like this comment didn't get included in the last review, so here it goes again: Would "client" make more sense?

The server is getting installed either way. The description for dedicated seems to indicate whether to install a server.

Reply:
`dedicated` is a set precedent as a USE flag among ebuilds such as this, actually. I have encountered it numerous times. For consistency I felt it was natural to adhere to using this flag in this manner. Another reason that it's favourable to use `dedicated` rather than `client` is that it's an option that's off by default, and most users installing it will likely opt to leave it off, whereas with client, such a flag would not be enabled by default in make.conf or in the system profile, and would require enabling as it's probably the most common way to install this.

Reply:
Okay, then you should locally define `dedicated`.

games-fps/doom3 locally defines it as: install only as a dedicated game server

This would make it clearer as what the outcome will be when the flag is enabled.

Reply:
There is an existing description for this flag, but it's not as good as the one you cited from doom3's ebuild, so I've made the change accordingly.